src/com/twu/biblioteca/Book.java

```diff
@@ -0,0 +1,35 @@
+package com.twu.biblioteca;
+
+public class Book {
+    private final String title;
+    private final String author;
+    private final int yearPublished;
+    private boolean checkedOut;
+
+    public Book(String title, String author, int yearPublished, boolean checkedOut) {
+        this.title = title;
+        this.author = author;
+        this.yearPublished = yearPublished;
+        this.checkedOut = false;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public String getAuthor() {
+     return author;
+   }
+
+   public int getYearPublished() {
+      return yearPublished;
+   }
+
+   public boolean getCheckedOut() {
```

Comment:
From the Java spec around property getters:
```
8.3.2 Boolean properties

In addition, for boolean properties, we allow a getter method to match the pattern:

public boolean is<PropertyName>();

This “is<PropertyName>” method may be provided instead of a “get<PropertyName>” method, or it may be provided in addition to a “get<PropertyName>” method. 
```